bin/setup

```diff
@@ -10,13 +10,15 @@ chdir APP_ROOT do
   step 'Installing dependencies' do
     system! 'gem install bundler --conservative'
     system('bundle check') || system!('bundle install')
+    system('which direnv') || system!('brew install direnv')
   end
 
   step 'Copying sample files' do
     ['.envrc'].each do |file|
       unless File.exist?(file)
         cp "#{file}.sample", file
         puts "Copied #{file}.sample to #{file}"
+        system! "ln -s #{file} .env"
```

Comment:
question for Citizen Code folks: why do we have an `envrc.sample` file rather than `env` ?